src/gui/transferlistmodel.cpp

```diff
@@ -727,55 +727,55 @@ QColor getDefaultColorByState(const BitTorrent::TorrentState state)
 
 QIcon getPausedIcon()
 {
-    static QIcon cached = QIcon(":/icons/paused.svg");
+    static QIcon cached = UIThemeManager::instance()->getIcon(QLatin1String("paused"));
```

Comment:
Are all these caching wrappers still needed? IIRC, manager has icon cache.

Reply:
saves hash lookups, but idk I'll remove it if you say so.

Reply:
>saves hash lookups

Ok. Seems they're accessed frequently so it makes sense.